Reversi/Reversi/Module1.vb

```diff
@@ -4,7 +4,20 @@
         Public column As Integer
     End Class
 
-    Function CheckPlayerMove(ByRef Board(,) As Char, ByVal row As Integer, ByVal column As Integer, ByRef items(,) As Integer) As List(Of Field)
+    Sub FlipPieces(ByRef piecesFlipped As List(Of Field), ByRef Board(,) As Char, ByVal colour As Char)
+
+        For Each field In piecesFlipped
+
+            If colour = "W" Then
+                Board(field.row, field.column) = "W"
+            Else
+                Board(field.row, field.column) = "B"
+            End If
+        Next
+
+    End Sub
+
+    Function CheckMove(ByRef Board(,) As Char, ByVal row As Integer, ByVal column As Integer, ByRef items(,) As Integer, ByVal colour As Char) As List(Of Field)
```

Comment:
Add an enum type (i.e ReversiPawn) with two values black white and empty. Remove argument color and replace it with the variable of previously defined enum type